src/crypto/algorithms/megolm.ts

```diff
@@ -1424,17 +1426,17 @@ class MegolmDecryption extends DecryptionAlgorithm {
      *
      * @param {module:models/event.MatrixEvent} event key event
      */
-    public onRoomKeyEvent(event: MatrixEvent): Promise<void> {
-        const content = event.getContent();
-        const sessionId = content.session_id;
+    public async onRoomKeyEvent(event: MatrixEvent): Promise<void> {
+        const content: Partial<IMessage["content"]> = event.getContent();
```

Comment:
Can we use the generic signature instead?

```suggestion
        const content = event.getContent<Partial<IMessage["content"]>>();
```